content/concept-accounts.md

```diff
@@ -0,0 +1,123 @@
+# RCL Accounts
+
+An "Account" in the Ripple Consensus Ledger represents a holder of XRP and a sender of transactions. The core elements of an account are:
+
+- An identifying **address**, such as `rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn`
+- An account can send [transactions](reference-transaction-format.html) to modify the Ripple Consensus Ledger, for a variety of purposes including:
```

Comment:
You're right that it didn't fit in. Ultimately I think a better bit of information to capture in this section is the unit(s) of authentication the account contains, rather than the types of transactions it can send, so I've replaced this bullet point with the following:

> - One or more ways to [authorize transactions](reference-transaction-format.html#authorizing-transactions), possibly including:
>    - A master key pair intrinsic to the account. (This can be [disabled](reference-transaction-format.html#accountset-flags) but not changed.)
>    - A "regular" key pair that [can be rotated](reference-transaction-format.html#setregularkey).
>    - A signer list for [multi-signing](reference-transaction-format.html#multi-signing). (Stored separately from the account's core data.)